client/src/components/message/MessageContent.js

```diff
@@ -40,7 +40,10 @@ class MessageContent extends Component {
       code: ({text}, k) => rich ? <code key={k}>{text}</code> : text,
       emoji: ({id}, k) => <Emoji name={id} customEmojis={customEmojis} key={k}/>,
       user: ({id}, k) => {
-        const name = `@${users[id].display_name}`
+        if (!users[id]) {
+          return `@${id}`
+        }
+        const name = `@${users[id].display_name || users[id].real_name}`
```

Comment:
This is not related to the crash, but some users have `display_name === ""`, so only an `@` appeared.

Reply:
Is it okay to do this?

Reply:
To fix something not related to the issue being solved?
Or to display one's real name if they haven't set a display name?

Reply:
:laughing: 

> To fix something not related to the issue being solved?

I mean, I would prefer if you mention it in the commit message, but okay. :smile: 

> Or to display one's real name if they haven't set a display name?

I was interested in this one.

Reply:
Probably, this should work without a problem.

Reply:
You can check anyone's real name on slack, so I don't see any problem with displaying it in the reports.